roles/ceph-mon/defaults/main.yml

```diff
@@ -42,16 +41,16 @@ calamari: false
 openstack_config: false
 openstack_glance_pool:
   name: images
-  pg_num: "{{ pool_default_pg_num }}"
+  pg_num: "{{ osd_pool_default_pg_num }}"
 openstack_cinder_pool:
   name: volumes
-  pg_num: "{{ pool_default_pg_num }}"
+  pg_num: "{{ osd_pool_default_pg_num }}"
 openstack_nova_pool:
   name: vms
-  pg_num: "{{ pool_default_pg_num }}"
+  pg_num: "{{ osd_pool_default_pg_num }}"
 openstack_cinder_backup_pool:
   name: backups
-  pg_num: "{{ pool_default_pg_num }}"
+  pg_num: "{{ osd_pool_default_pg_num }}"
```

Comment:
It shouldn't, at least it does not since we check variables (see below), I'm testing this locally.